src/renderer/lib/dolphin/useDolphinActions.ts

```diff
@@ -51,6 +51,22 @@ export const useDolphinActions = (dolphinService: DolphinService) => {
     [dolphinService, showError],
   );
 
+  const getDolphinVersion = useCallback(
+    async (dolphinType: DolphinLaunchType): Promise<string> => {
+      try {
+        const version = await dolphinService.getDolphinVersion(dolphinType);
+        const rtn: string = version["result"]["version"];
+        if (rtn === "") {
+          return "Unkown";
+        }
+        return rtn;
+      } catch (err) {
+        return "Unkown";
```

Comment:
```suggestion
        return "";
```
I'd rather the individual component choose to handle rendering an undefined or empty string.